modules/ROOT/pages/comments-commands-events-apis.adoc

```diff
@@ -10,3 +10,60 @@
 The Comments plugin provides the following {productname} commands.
 
 include::partial$commands/comments-cmds.adoc[]
+
+== Events
+
+include::partial$events/tinycomments-events.adoc[]
+
+=== CommentChange event
+
+include::partial$misc/admon-requires-6.1v.adoc[]
+
+Fired when a comment is added, resolved, deleted, edited, or replied to. Contains the `getEventLog` API for retrieving log of comment changes.
+
+==== Example: The CommentChange event
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'tinycomments',
+  toolbar: 'addcomment showcomments',
+  init_instance_callback: (editor) => {
+    editor.on('CommentChange', (evt) => {
+      console.log(evt.getEventLog());
+    });
+  }
+});
+----
+
+== APIs
+
+The {pluginname} plugin provides the following API.
+
+[[getEventLog]]
+=== getEventLog()
+
+include::partial$misc/admon-requires-6.1v.adoc[]
+
+Returns a log with information and timestamps of all changes to comments, including when:
+
+* A new comment is added (and who added it).
+* A comment is edited (and who edited it).
+* A reply to a comment is added (and who added the reply).
+* A comment is resolved (and who resolved the comment).
+* A comment is deleted (and who deleted the comment).
+
+The event log can be retrieved either in full or with the `after` option, which restricts the returned list to Comment events after a time-stamp date in the link:https://en.wikipedia.org/wiki/ISO_8601[ISO-8601] format, both shown in the following:
+
+==== Example: Using `+getEventLog()+`
+
+[source,js]
+----
+const comments = tinymce.activeEditor.plugins.tinycomments;
+
+console.log(comments.getEventLog());
+console.log(comments.getEventLog(
+    { after: '2022-02-22T12:34:56Z' }  //ISO-8601 standard: YYYY-MM-DDThh:mm:ssZ
```

Comment:
This doesn't follow our syntax style, it should be 2 spaces and there should be a space after the inline comment syntax like I had in my previous suggestion.
```suggestion
  { after: '2022-02-22T12:34:56Z' }  // ISO-8601 standard: YYYY-MM-DDThh:mm:ssZ
```